src/modules/fundingManager/bondingCurveFundingManager/RedeemingBondingCurveFundingManagerBase.sol

```diff
@@ -206,8 +206,11 @@ abstract contract RedeemingBondingCurveFundingManagerBase is
         if (_fee > BPS) {
             revert RedeemingBondingCurveFundingManager__InvalidFeePercentage();
         }
-        emit SellFeeUpdated(_fee, sellFee);
+        uint oldSellFee = sellFee;
```

Comment:
Why did you change that?

Reply:
The event was misplaced on my branch and returning the same number twice. I agree emitting before is the more elegant solution. fixed